phix.html.markdown

```diff
@@ -0,0 +1,454 @@
+---
+language: Phix
+contributors:
+    - ["pxtom", "https://gitgub.com/pxtom"]
+filename: learnphix.exw
+---
+
+
+            -- single line comment
+
+            // single line comment
+
+            /* multi-
+                     line comment */
+
+// Start programming immediately
+
+    -- write using UTF8; save as: hello.ex
+    -- use ? for output
+
+        ? "😍 hello , 😎 world!"
+        ? sqrt(2+2)
+
+// Interpret your program
+                            /*
+       p hello               */
+
+// Compile your program
+                            /*
+       p -c hello            */
+
+// Coding mistakes receive gentle help messages
+                                                          /*
+    string line
+    line = 5
+           ^ type error (storing atom in string)           */
+
+// Every literal value, constant, and variable is an ''object''
+
+    -- a literal object
+        ? "hello"
+        ? PI
+        ? { "hello", PI }
+
+    -- a named variable object
+        object X
+        X = "hello"
+        X = PI
+        X = { "hello", PI }
+
+    -- a named constant object
+        constant myPI = 22/7
+
+//  Everything is an ''object'', just two fundemental kinds
+                                                                /*
```

Comment:
Markdown View is being displayed different than in source diff, It's showing in as single line comment instead of multiline comment.